src/display/editor/annotation_editor_layer.js

```diff
@@ -75,7 +75,7 @@ class AnnotationEditorLayer {
   constructor(options) {
     if (!AnnotationEditorLayer._l10nInitialized) {
       AnnotationEditorLayer._l10nInitialized = true;
```

Comment:
Also, maybe just call the property `_initialized` now since this is no longer just for l10n-handling?